code/datums/map_config.dm

```diff
@@ -29,6 +29,13 @@
 	var/ftl_ship_dheight = 62
 	var/ftl_ship_width = 73
 	var/ftl_ship_height = 113
+
+	var/fob_ship_dir = NORTH
+	var/fob_ship_dwidth = 15
+	var/fob_ship_dheight = 0
+	var/fob_ship_width = 22
+	var/fob_ship_height = 13
+

```

Comment:
This is the aetherwhisp's shuttle right?

Reply:
Yes. But I should rename fob_ship_ to fob_shuttle_